.prettierignore

```diff
@@ -0,0 +1,2 @@
+# Ignore vendor files:
+reset.css
```

Comment:
We should probably add `CODE_OF_CONDUCT.md` here, since it's also a "vendor" file in a sense.